app/assets/stylesheets/main.scss

```diff
@@ -82,6 +82,24 @@
   .grader_distribution {
     margin-bottom: 0;
   }
+
+  #grader-right{
+    height:380px;
+    overflow:scroll;
+
+  }
+
+  ::-webkit-scrollbar {
+    -webkit-appearance: none;
+    width: 7px;
+  }
+
+  ::-webkit-scrollbar-thumb {
+    border-radius: 4px;
+    background-color: rgba(0, 0, 0, .5);
+    -webkit-box-shadow: 0 0 1px rgba(255, 255, 255, .5);
```

Comment:
Color literals like `rgba(255, 255, 255, 0.5)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`.5` should be written with a leading zero as `0.5`